dist/aurora_apis.js

```diff
@@ -0,0 +1,17 @@
+//[aurora]
+// Uncomment the previous line for testing on webpagetest.org
+
+// Detects if NgOptimizedImage is in use on the page
+function isAngularImageDirUser() {
+  return !!document.querySelector('img[ng-img]');
+}
+
+// Detects count of NgOptimizedImage instances with a priority attr 
+function getAngularImagePriorityCount() {
+    return document.querySelectorAll('img[ng-img][priority]').length;
+}
+
+return JSON.stringify({
+    ng_img_user: isAngularImageDirUser(),
+    ng_priority_img_count: getAngularImagePriorityCount()
+});
```

Comment:
Older metrics required scalar outputs, but WPT can handle anything serializable including objects. This also simplifies how we query the results:

```suggestion
return {
    ng_img_user: isAngularImageDirUser(),
    ng_priority_img_count: getAngularImagePriorityCount()
};
```